libs/category/src/models/generic-category.ts

```diff
@@ -10,7 +11,7 @@ import { DaffCategoryBreadcrumb } from './category-breadcrumb';
  * in Daffodil. As Categories are often considered Trees of Categories,
  * this type is a recursive generic.
  */
-export interface DaffGenericCategory<T extends DaffGenericCategory<T>> extends DaffLocatable, DaffIdentifiable {
+export interface DaffGenericCategory<T extends DaffGenericCategory<T>> extends DaffLocatable, DaffIdentifiable, Partial<DaffCanonicallyLocatable> {
```

Comment:
why is this a `Partial`?

Reply:
I was just following the example of `meta_description`, etc. that seo fields should be optional. I don't have a rationale other than that.